### ISSUE TITLE
TypeError null is not an object (evaluating 'd.trackingVersion')

### ISSUE THREAD

**Issue by n19htz** [angular/angular#50989]
### Which @angular/* package(s) are the source of the bug?

core

### Is this a regression?

Yes

### Description

I can't reproduce error, but clients often getting this error:

<img width="1672" alt="Screenshot 2023-07-10 at 15 35 19" src="https://github.com/angular/angular/assets/39006390/bffd652f-9dfd-403c-b104-82dccb3266ff">

### Please provide a link to a minimal reproduction of the bug

-

### Please provide the exception or error you saw

```true
TypeError
null is not an object (evaluating 'd.trackingVersion')
```

### Please provide the environment you discovered this bug in (run `ng version`)

```true
Angular CLI: 16.1.4
Node: 20.4.0 (Unsupported)
Package Manager: npm 9.8.0
OS: darwin arm64

Angular: 16.1.4
... animations, cdk, cli, common, compiler, compiler-cli, core
... forms, language-service, material, material-date-fns-adapter
... platform-browser, platform-browser-dynamic, router

Package                         Version
---------------------------------------------------------
@angular-devkit/architect       0.1601.4
@angular-devkit/build-angular   16.1.4
@angular-devkit/core            16.1.4
@angular-devkit/schematics      16.1.4
@schematics/angular             16.1.4
ng-packagr                      16.1.0
rxjs                            7.8.1
typescript                      5.1.6
webpack                         5.88.1
```

**Comment by n19htz** [comment]
> What browser (and version) is used that report these errors?

![photo_2023-07-10 16 07 42](https://github.com/angular/angular/assets/39006390/0dcc0545-9005-4db8-8129-27b3cb23fc14)

This is latest error. All of those errors in sentry report are safari 15+, 16+ versions

**Comment by elgreco247** [comment]
Safari bug apparently was fixed with version 16.1: https://github.com/WebKit/WebKit/commit/44c15ba58912faab38b534fef909dd9e13e095e0
